package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "openmrs-owa-orderentry",
+  "version": "0.1.0",
+  "description": "It is designed to address a set of specific use cases – e.g., placing an order, revising it, discontinuing it, and being able to look up a patient&#39;s active orders.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/geofrocker/openmrs-owa-orderentry"
+  },
+  "dependencies": {
+    "coveralls": "^3.0.0",
+    "enzyme-adapter-react-15": "^1.0.5",
+    "react": "^15.4.1",
```

Comment:
Noticed you used React version 15. Considering that it is a new project, any reasons for using React 15 instead of the latest version which is 16?

Reply:
The [openmrs owa generator](https://www.npmjs.com/package/generator-openmrs-owa) creates the app using react 15. I did not see the reason to change it by then incase it has files files that depend on it specifically.

Reply:
Did you try upgrading the version and see what happens?

Reply:
Have updated. It only affected the tests. But have also upgraded to enzyme-adapter-react-16
Have also upgraded to react-router 4 as suggested by @larrystone

Reply:
Looks good.
Nice work upgrading the project and affected files.